integration-tests/utils/integration-helpers.js

```diff
@@ -22,9 +22,10 @@ const withSignedInPage = fn => async options => {
 
 const clipToken = str => str.toString().substr(-10, 10)
 
-const makeWorkspace = withSignedInPage(async ({ page, billingProject }) => {
-  const workspaceName = `test-workspace-${Math.floor(Math.random() * 100000)}`
+const testWorkspaceName = () => `test-workspace-${Math.floor(Math.random() * 100000)}`
```

Comment:
We typically use verbs for function names (and nouns for others) (e.g. `getTestWorkspaceName`) to make it easier to know at a glance if a name belongs to a function.